docker/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM registry.redhat.io/ubi9/ubi:9.5 as base
+
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV RH_CERT_URL="https://certs.corp.redhat.com"
```

Comment:
this is a internal URL, please move it for a internal only place (e.g. kubernetes configmap / github environments)